Sources/CMakeLists.txt

```diff
@@ -8,6 +8,48 @@ See http://swift.org/LICENSE.txt for license information
 See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 ]]
 
+include(CMakeParseArguments)
+function(SwiftBuild_Bundle)
+  set(Options)
+  set(OneValueArguments MODULE)
+  set(MultiValueArguments FILES)
+  cmake_parse_arguments(PARSE_ARGV 0 BundleXCSpecs
+    "${Options}" "${OneValueArguments}" "${MultiValueArguments}")
+
+  list(TRANSFORM BundleXCSpecs_FILES PREPEND "${CMAKE_CURRENT_SOURCE_DIR}/")
+  add_custom_command(TARGET ${BundleXCSpecs_MODULE} POST_BUILD
+    COMMAND
+      ${CMAKE_COMMAND} -E make_directory "${CMAKE_BINARY_DIR}/share/pm/SwiftBuild_${BundleXCSpecs_MODULE}.resources"
+    COMMAND
+      ${CMAKE_COMMAND} -E copy_if_different ${BundleXCSpecs_FILES} "${CMAKE_BINARY_DIR}/share/pm/SwiftBuild_${BundleXCSpecs_MODULE}.resources/")
+
+  file(TO_NATIVE_PATH "${CMAKE_BINARY_DIR}/share/pm/SwiftBuild_${BundleXCSpecs_MODULE}.resources" _SWIFT_BUILD_RESOURCE_BUNDLE_BUILD_PATH)
```

Comment:
I wonder if we can use `$<TARGET_PROEPRTY:${BundleXCSpecs_MODULE},BINARY_DIR>` to get the module's binary dir rather than placing this at the top-level.

Reply:
gave this a try but I was unable to use a generator expression in the output path when creating the resource accessor